src/search/search-container.jsx

```diff
@@ -40,15 +53,45 @@ export class SearchContainer extends React.Component {
         })
       })
       .catch(error => {
-        // TODO: Show user-friendly message
-        console.log(error)
         this.setState({
           error,
           isLoaded: true
         })
+
+        toast.error(
+          <div>
+            Unable to load movies:<br />
+            {error.toString()}
+          </div>,
+          {
+            position: toast.POSITION.BOTTOM_CENTER,
+            autoClose: false
+          }
+        )
       })
   }
 
+  reloadFilms() {
+    this.updateQueryParams()
+    this.loadFilms(this.queryParams.getParams())
+  }
+
+  updateQueryParams() {
+    this.setState({
+      queryParams: this.queryParams.getParams()
+    })
+  }
+
+  onSortByChange(sortBy) {
```

Comment:
You can define class field to avoid using `bind(this)` in the render method
`onSortByChange = (sortBy) => {`

Reply:
Are you going to split css file?

Reply:
> You can define class field to avoid using bind(this) in the render method
onSortByChange = (sortBy) => {
Yeah, I'll probably follow this style

> Are you going to split the css file?
Eventually yes. I'm planning to extract component specific styles to their own `styles.css`. I consider changing project structure